app/src/main/java/com/openclassrooms/realestatemanager/repository/EstateRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.openclassrooms.realestatemanager.repository
+
+import com.openclassrooms.realestatemanager.api.EstateApi
+import com.openclassrooms.realestatemanager.model.Estate
+
+class EstateRepository(val estateApi: EstateApi) {
+
+    fun getEstateList(): List<Estate> {
+        return estateApi.getEstateList()
+    }
```

Comment:
```suggestion
    fun getEstateList(): List<Estate> = estateApi.getEstateList()
```

Optionnel, c'est pour te montrer une syntaxe possible avec Kotlin :)

Reply:
Ok super, c'est pratique :)